src/POData/UriProcessor/QueryProcessor/OrderByParser/OrderByParser.php

```diff
@@ -116,6 +115,8 @@ public static function parseOrderByClause(
         ProvidersWrapper $providerWrapper
     ) {
         assert(is_string($orderBy), "OrderBy clause must be a string");
+        $orderBy = trim($orderBy);
+        assert(0 < strlen($orderBy), "OrderBy clause must not be trimmable to an empty string");
```

Comment:
Trap an empty $orderBy string here to prevent the buildOrderByTree call barfing, complaing about an empty array